ProcessLib/HT/Tests.cmake

```diff
@@ -349,6 +349,28 @@ AddTest(
     CoupledPressureParabolicTemperatureParabolic_ts_10_expected.vtu CoupledPressureParabolicTemperatureParabolicStaggered_pcs_1_ts_10_t_1.000000.vtu darcy_velocity darcy_velocity 1e-10 1e-10
 )
 
+AddTest(
+    NAME HT_SimpleSynthetics_constraint_dirichlet_bc
+    PATH Parabolic/HT/SimpleSynthetics
+    EXECUTABLE ogs
+    EXECUTABLE_ARGS constraint_bc_1e3.prj
+    TESTER vtkdiff
+    REQUIREMENTS NOT OGS_USE_MPI
+    DIFF_DATA
+    ConstViscosityThermalConvection_pcs_0_ts_1_t_0.000000.vtu ConstViscosityThermalConvection_pcs_0_ts_1_t_0.000000.vtu T T 1e-15 1e-14
+    ConstViscosityThermalConvection_pcs_0_ts_2_t_0.000010.vtu ConstViscosityThermalConvection_pcs_0_ts_2_t_0.000010.vtu T T 1e-15 1e-14
+    ConstViscosityThermalConvection_pcs_0_ts_3_t_0.001010.vtu ConstViscosityThermalConvection_pcs_0_ts_3_t_0.001010.vtu T T 1e-14 1e-13
+    ConstViscosityThermalConvection_pcs_0_ts_4_t_0.101010.vtu ConstViscosityThermalConvection_pcs_0_ts_4_t_0.101010.vtu T T 1e-14 1e-13
+    ConstViscosityThermalConvection_pcs_0_ts_5_t_1.101010.vtu ConstViscosityThermalConvection_pcs_0_ts_5_t_1.101010.vtu T T 1e-14 1e-13
+    ConstViscosityThermalConvection_pcs_0_ts_6_t_10.000000.vtu ConstViscosityThermalConvection_pcs_0_ts_6_t_10.000000.vtu T T 1e-14 1e-13
+    ConstViscosityThermalConvection_pcs_0_ts_1_t_0.000000.vtu ConstViscosityThermalConvection_pcs_0_ts_1_t_0.000000.vtu p p 1e-15 1e-14
+    ConstViscosityThermalConvection_pcs_0_ts_2_t_0.000010.vtu ConstViscosityThermalConvection_pcs_0_ts_2_t_0.000010.vtu p p 1e-15 1e-14
+    ConstViscosityThermalConvection_pcs_0_ts_3_t_0.001010.vtu ConstViscosityThermalConvection_pcs_0_ts_3_t_0.001010.vtu p p 1e-15 1e-14
+    ConstViscosityThermalConvection_pcs_0_ts_4_t_0.101010.vtu ConstViscosityThermalConvection_pcs_0_ts_4_t_0.101010.vtu p p 1e-15 1e-14
+    ConstViscosityThermalConvection_pcs_0_ts_5_t_1.101010.vtu ConstViscosityThermalConvection_pcs_0_ts_5_t_1.101010.vtu p p 1e-15 1e-14
+    ConstViscosityThermalConvection_pcs_0_ts_6_t_10.000000.vtu ConstViscosityThermalConvection_pcs_0_ts_6_t_10.000000.vtu p p 1e-15 1e-14
```

Comment:
Could be reduced to two lines using GLOBs. See https://github.com/ufz/ogs/blob/master/ProcessLib/RichardsComponentTransport/Tests.cmake#L10-L11